keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -2894,6 +2965,52 @@ type ResourceDriverFeature struct {
 }
 ```
 
+### Complex sharing of ResourceClaim
+
+At the moment, the allocation result marks as a claim as either "shareable" by
```

Comment:
There's something not quite right in that model. I wonder if the underlying semantic models can express this?

Also, there are different kinds of "shareable" - for example, I know of use cases where you want to mount the GPU devices in a monitoring container - or perhaps to access it via some config tools.

No change needed, just raising some things to think about post 1.30.